config/locales/en.yml

```diff
@@ -205,7 +205,7 @@ en:
           body_info: Include as much detail as you can. This will help us process your application more quickly.
       has_other_parties:
         edit:
-          page_title: Other parties required
+          page_title: Other parties who should know about application
```

Comment:
Is a `the` missing here? "about the application"

Reply:
I've just double checked and it's the recommendation Amanda's made and it's a direct copy and paste - is it worth us confirming with her when she's back?